otelcollector/deploy/addon-chart/azure-monitor-metrics-addon/templates/ama-metrics-daemonset.yaml

```diff
@@ -414,11 +425,22 @@ spec:
             - name: MINIMAL_INGESTION_PROFILE
               value: "true" # only supported value is the string "true"
             - name: APPMONITORING_AUTOINSTRUMENTATION_ENABLED
-              value: "{{ .Values.AzureMonitorMetrics.IsAppMonitoringAutoInstrumentationEnabled }}"
+              value: "{{ .Values.AppmonitoringAgent.enabled }}"
             - name: APPMONITORING_OPENTELEMETRYMETRICS_ENABLED
-              value: "{{ .Values.AzureMonitorMetrics.IsAppMonitoringOpenTelemetryMetricsEnabled }}"
+              value: "{{ .Values.AppmonitoringAgent.isOpenTelemetryMetricsEnabled | default false }}"
             - name: APPMONITORING_OPENTELEMETRYMETRICS_PORT
-              value: "{{ .Values.AzureMonitorMetrics.OpenTelemetryMetricsPort }}"
+              value: "{{ .Values.AppmonitoringAgent.openTelemetryMetricsPort | default 28333 }}"
+          {{- if .Values.AppmonitoringAgent.isOpenTelemetryMetricsEnabled | default false }}
+          ports:
+            - name: otlp-http-port
+              containerPort: 56681
+              hostPort: {{ .Values.AppmonitoringAgent.openTelemetryMetricsPort | default 28333 }}
+              protocol: TCP
+            - name: otlp-grpc-port
```

Comment:
need grpc now ? may be add this when grpc is fully supported ? this way, we we dont fail if this specific port is taken though its not used ?